db/migrate/20160429150155_alter_games_table_change_board_column_to_json.rb

```diff
@@ -0,0 +1,6 @@
+class AlterGamesTableChangeBoardColumnToJson < ActiveRecord::Migration
+  def change
+    remove_column(:games, :board)
+    add_column(:games, :board, :json)
```

Comment:
yeah rails takes care of converting it from and to json, so it should be fine

merging